cmd/get/matchable_task_resource_attribute.go

```diff
@@ -0,0 +1,101 @@
+package get
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/flyteorg/flytectl/cmd/config"
+	"github.com/flyteorg/flytectl/cmd/config/subcommand"
+	cmdCore "github.com/flyteorg/flytectl/cmd/core"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+)
+
+const (
+	taskResourceAttributesShort = "Gets matchable resources of task attributes"
+	taskResourceAttributesLong  = `
+Retrieves task  resource attributes for given project,domain combination or additionally with workflow name.
+
+Retrieves task resource attribute for project and domain
+Here the command get task resource attributes for  project flytectldemo and development domain.
+::
+
+ flytectl get task-resource-attribute -p flytectldemo -d development 
+
+eg : O/P
+
+.. code-block:: json
+
+ {"Project":"flytectldemo","Domain":"development","Workflow":"","defaults":{"cpu":"1","memory":"150Mi"},"limits":{"cpu":"2","memory":"450Mi"}}
+
+Writing the task resource attribute to a file. If there are no task resource attributes a file would be written with basic data populated.
+Here the command gets task resource attributes and writes the config file to tra.yaml
+eg:  content of tra.yaml
+
+::
+
+ flytectl get task-resource-attribute --attrFile tra.yaml
```

Comment:
will other gets to matchable resources also re-use the --attrFile output param? it would be nice for consistency!

Reply:
Yes thats right , We will have the same attrFile for other matchable entities aswell. 
May be we can rename it if it doesn't sound right.

Reply:
I think that is the case, currently only one is implemented